arduino/smartcar/smartcar.ino

```diff
@@ -100,16 +117,24 @@ void setup() {
 }
 
 void loop() {
+   handleInput();
+ {
+    Serial.println((leftOdometer.getDistance() + rightOdometer.getDistance())/2);
+
+	delay(5000);
```

Comment:
I've tested the code and it looks good, the only thing is the delay part which for some unknownn reason unables mqtt control and the obstacle avoidance in the loop.